the-collab-lab/tcl-21-smart-shopping-list
# #32. Styled Welcome Screen

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



1. Added a responsive logo using `react-tilt-js` as well as a dynamic background using `react-particles-js`
2. Refactored the layout using Material UI's `Grid` and `Card` components.

## Related Issue



Closes #32 

## Acceptance Criteria



* Reformatted welcome screen layout to be displayed in two columns (like a traditional sign-in/ sign-up view)
* Added details around how to do both creating a new list and joining a new list.
* Added logo to main header bar
* Updated buttons 
* Updated typography

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates


<img src="https://media.giphy.com/media/hpkrfLugEq1aEuDlpN/giphy.gif"
/>




## Testing Steps / QA Criteria



1. Navigate to branch `tbm-welcome-screen` using the `git checkout` command in your terminal
2. Pull down the remote branch using the command `git pull`
3. Install new dependencies using `npm install`
4. Run `npm start` to start the server and see the changes
